src/App.js

```diff
@@ -40,8 +43,21 @@ class App extends Component {
       if (isAuthorized) {
         injected.activate()
         .then((a) => {
-          store.setStore({ account: { address: a.account }, web3context: { library: { provider: a.provider } } })
```

Comment:
Can we obtain network ID from `a`? I'm worried about changing this line because this is a high impact area. Would prefer to set account and web3context as we do today, and add a check above this line rather than setting store conditionally/in two different places. If this is the best solution need to do end-to-end testing with at least one deposit to make sure this doesn't break anything.

Reply:
Do you mean that we shouldn't do this:`const web3 = new Web3(a.provider)` but rather see if the `getNetworkType()` is already in `a`? I can check

Reply:
Yeah, you're right - seems like we can use `chainId` or `networkVersion` in the logic instead

![image](https://user-images.githubusercontent.com/165846/96670246-9c257280-13aa-11eb-8cb0-71cc02b8e72f.png)

Reply:
nice! I think this would be a lower risk change. just want to use extra caution here because wallet connect/store stuff can be high risk. I definitely support a solution that leaves the existing wallet connect lines in-tact, and instead works around them a bit!